Error with Texarea field

Hello!

Used stack:
-- `Symfony: v7.3.5`
-- `Sylius: v2.1.7`
-- `Sylius CMS plugin: v1.1.4`

I am getting an error each time i use Textarea field inside `sylius admin` in `CMS plugin`.

In _local_ project: `CMS pages` section - there are conflicts between textarea from `content > language` and textarea from `teaser > language`.
When populating first textarea and scrolling to second, first text becomes not editable and if `update` is pressed - it becomes empty.
Can not get this behaviour on `v2.demo.sylius.com`, though errors are displayed in browser console.

There are 3 errors in browser's console

_when adding and using textarea_
- Error 1: shows on local project

_on the event of page update_
- Error 2: shows on "https://v2.demo.sylius.com/admin/pages/"
- Error 3: shows on "https://v2.demo.sylius.com/admin/pages/"


[ 1 ]
```
stimulus.js:2148 Error connecting controller

TypeError: Cannot read properties of null (reading 'querySelectorAll')
    at extended.countErrors (CompoundFormErrorsController.js:80:1)
    at CompoundFormErrorsController.js:67:1
    at NodeList.forEach (<anonymous>)
    at extended.updateErrorBadges (CompoundFormErrorsController.js:66:1)
    at extended.updateFormErrors (CompoundFormErrorsController.js:57:1)
    at extended.connect (CompoundFormErrorsController.js:29:1)
    at Context.connect (stimulus.js:1498:1)
    at Module.connectContextForScope (stimulus.js:1668:1)
    at stimulus.js:2067:1
    at Array.forEach (<anonymous>)

Object
controller
: 
extended {context: Context, tabErrorBadgeClass: 'tab-error', accordionErrorBadgeClass: 'accordion-error', observer: MutationObserver}
element
: 
form#sylius_cms_admin_page.ui.loadable.form
identifier
: 
"sylius--admin-bundle--compound-form-errors"
[[Prototype]]
: 
Object
```

[ 2 ]
```
admin-entry.dde61666.js:2 Error connecting controller

TypeError: Cannot read properties of null (reading 'querySelectorAll')
    at t.countErrors (569.3d99199b.js:1:1388)
    at 569.3d99199b.js:1:1176
    at NodeList.forEach (<anonymous>)
    at t.updateErrorBadges (569.3d99199b.js:1:1150)
    at t.updateFormErrors (569.3d99199b.js:1:931)
    at t.connect (569.3d99199b.js:1:365)
    at F.connect (admin-entry.dde61666.js:2:609845)
    at B.connectContextForScope (admin-entry.dde61666.js:2:612647)
    at admin-entry.dde61666.js:2:620237
    at Array.forEach (<anonymous>)

{identifier: 'sylius--admin-bundle--compound-form-errors', controller: t, element: form#sylius_cms_admin_page.ui.loadable.form}
controller
: 
t {context: F, tabErrorBadgeClass: 'tab-error', accordionErrorBadgeClass: 'accordion-error', observer: MutationObserver}
element
: 
form#sylius_cms_admin_page.ui.loadable.form
identifier
: 
"sylius--admin-bundle--compound-form-errors"
[[Prototype]]
: 
Object
```

[ 3 ]
```
app-admin-entry.d36228b2.js:2 Uncaught TypeError: Cannot read properties of null (reading 'addEventListener')
    at new t (app-admin-entry.d36228b2.js:2:1162)
    at 4 (app-admin-entry.d36228b2.js:2:731)
    at n (app-admin-entry.d36228b2.js:2:108747)
    at app-admin-entry.d36228b2.js:2:388464
    at app-admin-entry.d36228b2.js:2:388470
    at app-admin-entry.d36228b2.js:2:388474
```

--- a/.github/workflows/build.yaml
+++ b/.github/workflows/build.yaml
@@ -79,10 +79,6 @@ jobs:
                 run: php -v | head -n 1 | awk '{ print $2 }' > .php-version
 
             -
-                name: Install certificates
-                run: symfony server:ca:install
-
-            -
                 name: Get Composer cache directory
                 id: composer-cache
                 run: echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT

--- a/behat.yaml.dist
+++ b/behat.yaml.dist
@@ -13,7 +13,7 @@ default:
 
         Behat\MinkExtension:
             files_path: "%paths.base%/vendor/sylius/sylius/src/Sylius/Behat/Resources/fixtures/"
-            base_url: "https://127.0.0.1:8080/"
+            base_url: "http://127.0.0.1:8080/"
             default_session: symfony
             javascript_session: chrome
             sessions:

--- a/templates/admin/shared/component_elements/form_theme.html.twig
+++ b/templates/admin/shared/component_elements/form_theme.html.twig
@@ -24,8 +24,8 @@
 
 {% block add_button_row %}
     {% if types is not empty %}
-        <div class="dropdown" {{ sylius_test_html_attribute('add-element-button') }}>
-            <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown">{{ label|trans }}</button>
+        <div class="dropdown" data-bs-toggle="dropdown" {{ sylius_test_html_attribute('add-element-button') }}>
+            <button class="btn dropdown-toggle" type="button">{{ label|trans }}</button>
             <ul class="dropdown-menu">
                 {% for type, label in types %}
                     <li>

